src/test/java/pl/piekoszek/gorskimatches/controllers/AccountControllerTest.java

```diff
@@ -0,0 +1,178 @@
+package pl.piekoszek.gorskimatches.controllers;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import pl.piekoszek.gorskimatches.token.TokenService;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class AccountControllerTest {
+
+    private final String email = "gorskimatchesserver@gmail.com";
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private TokenService tokenService;
+
+    @Test
+    public void shouldSendEmailAndReturnStringWhenEmailIsInCorrectFormat() throws Exception {
```

Comment:
It doesn't rly check is email was sent.
We talked about it -> https://discord.com/channels/960860284006846504/960864234118455316/1047977122825769000

Reply:
But on the other hand it's not super easy, so just create JIRA ticket for it ;)